controllers/notes.controller.js

```diff
@@ -0,0 +1,63 @@
+const { Note } = require('../models/note.model');
+const { extend } = require('lodash');
+
+const createNoteForVideo = async (req, res) => {
+	try {
+		let newNote = req.body;
+		newNote = new Note(newNote);
+		newNote = await newNote.save();
+		res.status(201).json({ response: newNote, success: true });
```

Comment:
I think here using the response keyword should be avoided, because API call will also return a response hence sometimes it might not be difficult to distinguish. would like to hear your intention as well

Reply:
I wanted to make the API consistent, That's why used response everywhere. Will think of any other name if it's confusing.